### ISSUE TITLE
fullscreen:1 not workings as expeted on macOS

### ISSUE THREAD

**Issue by sorenpeter** [rumblesan/improviz#48]
When I set the "fullscreen: 1" in config.yaml, the windows on my 2nd screen is taking up alle the space, but I still need go to the Window-menu and select Enten Full Screen to get the resolution right. If not the code-overlay is too big.

**Comment by rumblesan** [comment]
hey, not sure I quite get what you mean sorry. What OS are you using this on? And is it possible to get some screen shots to demonstrate what you mean?

**Comment by sorenpeter** [comment]
Hey, here are some screenshots.
![Screenshot 2020-02-15 at 10 28 52 (2)](https://user-images.githubusercontent.com/8458421/74585624-bcf92500-4fde-11ea-9474-a61b96219f81.png)
![Screenshot 2020-02-15 at 10 29 01 (2)](https://user-images.githubusercontent.com/8458421/74585625-bd91bb80-4fde-11ea-900e-93ffe8812de1.png)
![Screenshot 2020-02-15 at 10 29 06 (2)](https://user-images.githubusercontent.com/8458421/74585626-be2a5200-4fde-11ea-997c-7cd32b198ce6.png)

**Comment by sorenpeter** [comment]
Also if I excecute the code before I put the window in full screen then the font is way bigger than if i first exececlute the code:
![Screenshot 2020-02-15 at 11 04 59 (2)](https://user-images.githubusercontent.com/8458421/74585997-30049a80-4fe3-11ea-973e-339327cd19ee.png)
![Screenshot 2020-02-15 at 11 05 36 (2)](https://user-images.githubusercontent.com/8458421/74585998-3135c780-4fe3-11ea-83e3-8847a785d27a.png)

**Comment by rumblesan** [comment]
does your mac have a high DPI retina display? I can certainly imagine there's something weird happening there. I'll try to dig out a second monitor from somewhere so I can test this out.

**Comment by sorenpeter** [comment]
Yes my macbook have an internal retina display, but my external screen is not a high DPI one.

**Comment by sorenpeter** [comment]
Hey, This seem to also happen when decorated is set to true, as it is in the latest improviz-osx-0.7.1-editor-fix
<img width="1440" alt="Screenshot 2020-03-15 at 21 24 11" src="https://user-images.githubusercontent.com/8458421/76709966-94f70180-6703-11ea-8a8d-01782be7aae6.png">

**Comment by rumblesan** [comment]
so I've now got an external monitor, and I'm seeing the same thing. Does seem to be retina related so I'll start digging into it.
